packages/driver/src/util/config.ts

```diff
@@ -60,3 +72,57 @@ export const preprocessConfig = (config: Cypress.Config) => {
 export const preprocessEnv = (env: Cypress.ObjectLike) => {
   return preprocessForSerialization(env) as Cypress.Config
 }
+
+export const getMochaOverrideLevel = (state): MochaOverrideLevel | undefined => {
+  const test = state('test')
+
+  if (!state('duringUserTestExecution') && test && !Object.keys(test?._fired || {}).length) {
+    return test._testConfig.applied // either suite or test
+  }
+
+  return undefined
+}
+
+// Configuration can be override at multiple run-time levels. Ensure the configuration keys can
+// be override and that the provided override values are the correct type.
+//
+// The run-time override levels (listed in order applied):
+//   fileLoad -  config override via Cypress.config() when either loading the supportFile or specFile in the
+//                 browser (this is before mocha as process the spec
+//   suite    - config override via describe('', {...}, () => {})
+//   test     - config override via it('', {...}, () => {})
+//   event    - config override via Cypress.config() in test:before:runner or test:before:runner:async event
+//   runtime  - config override via Cypress.config() when the test callback is executed
```

Comment:
This comment should be updated

Reply:
I left it initially to document the order these overrides are applied since the aren't documented and not easy to find in the code base without stepping through the runner code.